cmd/secret/cmd/root_test.go

```diff
@@ -89,47 +115,72 @@ func TestRoot_Set(t *testing.T) {
 }
 
 func TestRoot_Get(t *testing.T) {
-	key := "key value"
+	key := "key-value"
 	value := "60OBdPOOkSOu6kn8ZuMuXtAPVrUEFkPREydDwY6+ip/LrAFaHSc="
 	path := "testFile.txt"
-	file, err := os.Create(path)
-	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
-	defer cancel()
+	t.Run("success", func(t *testing.T) {
+		file, err := os.Create(path)
+		ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
+		defer cancel()
 
-	require.NoError(t, err)
-	defer func() {
-		require.NoError(t, os.Remove(path))
-	}()
+		require.NoError(t, err)
+		defer func() {
+			require.NoError(t, os.Remove(path))
+		}()
 
-	defer func() {
-		require.NoError(t, file.Close())
-	}()
+		defer func() {
+			require.NoError(t, file.Close())
+		}()
 
-	fileTestData := make(map[string]string)
-	fileTestData[key] = value
-	require.NoError(t, json.NewEncoder(file).Encode(&fileTestData))
+		fileTestData := make(map[string]string)
+		fileTestData[key] = value
+		require.NoError(t, json.NewEncoder(file).Encode(&fileTestData))
 
-	r := New()
-	r.cmd.SetArgs([]string{"get", "--key", key, "--cipher-key", "ck", "--path", path})
-	executeErr := r.Execute(ctx)
-	require.NoError(t, executeErr)
+		r := New()
+		r.cmd.SetArgs([]string{"get", "--key", key, "--cipher-key", "ck", "--path", path})
+		executeErr := r.Execute(ctx)
+		require.NoError(t, executeErr)
 
-	testFile, err := os.Open(path)
-	require.NoError(t, err)
+		testFile, err := os.Open(path)
+		require.NoError(t, err)
 
-	defer func() {
-		require.NoError(t, testFile.Close())
-	}()
+		defer func() {
+			require.NoError(t, testFile.Close())
+		}()
 
-	fileData := make(map[string]string)
-	require.NoError(t, json.NewDecoder(testFile).Decode(&fileData))
-	var got string
-	require.Len(t, fileData, 1)
-	for _, value := range fileData {
-		got = value
-		break // we iterate one time to get first value
-	}
-	require.EqualValues(t, value, got)
+		fileData := make(map[string]string)
+		require.NoError(t, json.NewDecoder(testFile).Decode(&fileData))
+		var got string
+		require.Len(t, fileData, 1)
+		for _, value := range fileData {
+			got = value
+			break // we iterate one time to get first value
+		}
+		require.EqualValues(t, value, got)
+	})
+	t.Run("success set and read data from redis storage", func(t *testing.T) {
+		path := ""
+		redisURL := "localhost:6379"
+		ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
+		defer cancel()
+
+		r := New()
+		r.cmd.SetArgs([]string{"set", "--key", key, "--value", value, "--cipher-key", "ck", "--redis-url", redisURL})
+		err := r.Execute(ctx)
+		require.NoError(t, err)
+
+		_, err = os.Open(path)
+		require.Error(t, err)
+
+		rdb := redis.NewClient(&redis.Options{Addr: redisURL, Password: "", DB: 0})
+		s := storage.NewRedisVault(rdb)
+		err = s.SaveData([]byte(key), []byte(value))
+		require.NoError(t, err)
+
+		val, err := s.ReadData([]byte(key))
+		require.NoError(t, err)
+		require.EqualValues(t, value, val)
```

Comment:
Test is called TestRoot_Get, but you only set data from cli, but it should be set and get.